AmplifyTools/AmplifyXcode/Sources/AmplifyXcodeCore/Commands/CommandImportModels.swift

```diff
@@ -13,11 +13,18 @@ enum CommandImportModelsTasks {
                                           args: CommandImportModels.TaskArgs) -> AmplifyCommandTaskResult {
         let modelsPath = environment.path(for: args.generatedModelsPath)
         guard environment.directoryExists(atPath: modelsPath) else {
-            return .failure(
-                AmplifyCommandError(
-                    .folderNotFound,
-                    errorDescription: "Amplify generated models not found at \(modelsPath)",
-                    recoverySuggestion: "Run amplify codegen models."))
+            do {
+                _ = try environment.createDirectory(atPath: args.generatedModelsPath)
```

Comment:
curious why doesn't the directory exist on this particular `amplify pull`? how was it created before?